### DISCUSSION TITLE
Is Nitro supposed to reload on /api file changes?

### DISCUSSION THREAD

**Discussion by technicallypete** [nuxt/nuxt#18152] [Questions]
..Because it doesn't.

Fresh install and setup of nuxt3 boilerplate + vite + docker, everything works correctly, but updating ~/server/api/hello.ts doesn't reload anything, and the endpoint data never changes.

Confirmed that this is _not _ a docker mount issue.

Is an auto-reload feature for Nitro just missing right now? Or is this a bug we should file?

UPDATE: Just noticed this in the docs (in light grey font, lol): "The server directory is used to create any backend logic for your Nuxt application. It supports HMR and powerful features."

So i guess this should be working, but it isn't in my setup.. I'll jam on it more, but post a comment/answer if you're experiencing the same issue.

**Comment by justwiebe** [comment]
Using the bridge I haven't been able to get HMR working at all yet
